README.md

```diff
@@ -7,3 +7,56 @@
 ```
 npm install @hashmark/parser
 ```
+
+## What is Hashmark?
+Hashmark is a markup language. It's more structured than Markdown, but less verbose than XML. Here's what this README would look like in Hashmark:
+
+```hashmark
+#document Hashmark parser
+	#section Installation
+		#code bash
+			npm install @hashmark/parser
+
+	#section What is Hasmark?
+		Hashmark is a markup language. It's more structured than Markdown, ...
+```
+
+You can read more about the Hashmark language [in the docs](docs/what-is-hashmark.md).
+
+## Usage
+To parse a Hashmark document, 
+
+```typescript
+import { parse, ParsedSchema } from "@hashmark/parser";
+
+const file = "#tag example";
+const schemaFile = "#block tag"
+const schemaSchemaFile = "#block block";
+
+// Parse with a schema:
+const schemaSchema = new ParsedSchema(parse(schemaSchemaFile));
+const schema = new ParsedSchema(parse(schemaFile, schemaSchemaFile));
```

Comment:
Should be `schemaSchema` instead of `schemaSchemaFile`.

Reply:
Good catch! Fixed in 44c88ae81aa5537b9959f193838c612955c82195